arwen/elrondapi/elrondei.go

```diff
@@ -520,6 +521,12 @@ func v1_3_isSmartContract(context unsafe.Pointer, addressOffset int32) int32 {
 	}
 
 	isSmartContract := blockchain.IsSmartContract(address)
+
+	if !isSmartContract {
```

Comment:
this code is not needed. The IsSmartContract feature is not working properly on the WorldMock - in case of Elrond - blockchain hook it only verifies the type of the address. The worldMock makes something else.
The current code would return true even for a normal address which had a storage update.